test/inductor/test_cpu_repro.py

```diff
@@ -266,11 +267,15 @@ def forward(self, input_tensor):
 
         with torch.no_grad():
             compiled_m = torch.compile(m)
-            with self.assertRaisesRegex(
-                RuntimeError,
-                "output padding must be smaller than either stride or dilation",
-            ):
-                compiled_m(input)
+            if config.cpp_wrapper:
+                with self.assertRaises(RuntimeError):
```

Comment:
Is there an error msg pattern to check?

Reply:
Added one. It was a bit of a tossup, since most of the error message gets output to `stderr` and then lost from the `RuntimeError`, but there was still enough to make it work.